src/instruments/src/SD/StatusArea/StatusArea.jsx

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
This one still needs the G LOAD display so it's not a regression.

Reply:
This is not used yet.